cli/util/convert.go

```diff
@@ -64,7 +66,8 @@ is set to the target transaction's NetworkFee value plus one (if it's sufficient
 conflicting transaction itself). If the target transaction is not in the memory pool, standard 
 NetworkFee calculations are performed based on the calculatenetworkfee RPC request. If the --gas 
 flag is included, the specified value is added to the resulting conflicting transaction network fee 
-in both scenarios.
+in both scenarios. When the --await flag is included, the command waits for the conflicting 
+transaction to be included in a block before exiting and for the target transaction to be removed.
```

Comment:
the command waits for one of the conflicting or main transactions to be included in a block.